src/actions/guides/database/migrations.cr

```diff
@@ -400,6 +402,12 @@ class Guides::Database::Migrations < GuideAction
     end
     ```
 
+    This will generate a column called `author_id` on the `comments` table with a foreign key constraint pointing to an entry in the `users` table. It will also ensure that when a `User` is removed from the database, all associated `Comments` are also removed:
+
+    ```
+    "comments_author_id_fkey" FOREIGN KEY (author_id) REFERENCES users(id) ON DELETE CASCADE
+    ```
```

Comment:
Nice! Maybe you could split it into different lines so it doesn't overflow?

```suggestion

    ```
    comments_author_id_fkey" FOREIGN KEY (author_id) 
        REFERENCES users(id) ON DELETE CASCADE
    ```
```